app/flatpak-transaction.c

```diff
@@ -51,6 +51,7 @@ struct FlatpakTransactionOp {
 struct FlatpakTransaction {
   FlatpakDir *dir;
   GHashTable *refs;
+  GHashTable *remote_states;
```

Comment:
Would be good to clarify the type like  `GHashTable *remote_states;  /* (element-type utf8 FlatpakRemoteState) */`.